R/model-design-tools.R

```diff
@@ -1,39 +1,48 @@
-#' @title FUNCTION_TITLE
-#' @description FUNCTION_DESCRIPTION
-#' @param x PARAM_DESCRIPTION
-#' @return OUTPUT_DESCRIPTION
+#' @title Check a vector is a Date
+#' @description Checks that a vector is a date
+#' @param x A vector
+#' @return A logical
 #' @family utils
 is.Date <- function(x) {
   # nolint
   inherits(x, "Date")
 }
 
-#' @title FUNCTION_TITLE
-#' @description FUNCTION_DESCRIPTION
-#' @param data PARAM_DESCRIPTION
-#' @return OUTPUT_DESCRIPTION
+#' @title Convert all Dates to Factors
+#' @description Converts all Date columns to factors
+#' in a `data.frame`.
+#' @param data A `data.frame`.
```

Comment:
Yes we don't want to force data.table on people (though the output all being `data.table` does do that to some extent at the moment. Agree this can be added to the contributor guide but that feels like its own issue.

Reply:
Resolving here. Should move to an issue.